livekit_egress.proto

```diff
@@ -234,3 +236,10 @@ message FileInfo {
   int64 size = 4;
   string location = 5;
 }
+
+message SegmentsInfo {
+  string playlist_filename = 1;
```

Comment:
playlist_name to match SegmentedFileOutput

Reply:
Good idea.